File: services/utils-service/src/main/java/com/sgpttt/UtilsService/entity/Career.java
package com.sgpttt.UtilsService.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class Career {

    @Id
    @GeneratedValue(strategy=GenerationType.AUTO)
    @Column(name="career_id")
    private Long careerId;

    private String name;

    // Getters and Setters
    public Long getCareerId() {
        return careerId;
    }

    public void setCareerId(Long careerId) {
        this.careerId = careerId;
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
}

File: services/auth-service/src/main/kotlin/com/sgpttt/authservice/repository/entity/Career.kt
package com.sgpttt.authservice.repository.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
data class Career(
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	@Column(name = "career_id")
	val careerId: Long,
	
	val name: String
)